hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -124,7 +133,12 @@ public enum Counters {
       BADROWS,
       ONLY_IN_SOURCE_TABLE_ROWS,
       ONLY_IN_PEER_TABLE_ROWS,
-      CONTENT_DIFFERENT_ROWS
+      CONTENT_DIFFERENT_ROWS,
+      RE_COMPARES,
+      REJECTED_RE_COMPARES,
+      SOURCE_ROW_CHANGED,
+      PEER_ROW_CHANGED,
+      FAILED_RECOMPARE
```

Comment:
super nit: this is RECOMPARE and the others are RE_COMPARE

Reply:
can you change MAIN_THREAD_RE_COMPARES to MAIN_THREAD_RECOMPARES so it matches these? sorry to be pedantic haha